### PR TITLE
moved validate_forecast to its own file

### LINKED ISSUE #206: tidy: refactor move validate_forecast out to its own function
## Detailed Description
move validate_forecast out to its own function

## Context
- good to keep code tidy

## Possible Implementation
Move the code [here](move validate_forecast out to its own function) to its own file, perhaps called `validate_forecast.py`
Move tests too to reflect this change

### PR DESCRIPTION
**Pull request by zaryab-ali**
## Description
created a validate_forecast.py file and moved the validate_forecast function to it, and made the change in test file to import the function from the new file
Fixes #206

### CODE REVIEW

**Comment by zaryab-ali** [comment]
@peterdudfield please have a look

**Comment by peterdudfield** [comment]
Thanks @zaryab-ali, im gona merge into a dev branch to check tests are still wokring e.t.c

File: tests/test_forecast_compiler.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import os
 import pytest
 
-from pvnet_app.forecast_compiler import validate_forecast
+from pvnet_app.validate_forecast import validate_forecast
```
Comment (peterdudfield) [inline tests/test_forecast_compiler.py:7]: would you be able to move the tests that involve `validate_forecast` into a separate test file. Like test_validate_forecast.py. This keeps things nice and neat

Comment (zaryab-ali) [reply tests/test_forecast_compiler.py:7]: since all the functions currently in `test_forecast_compiler.py` just test `validate_forecast` with different edge cases, would it be ok if we just renamed the file to `test_validate_forecast.py` and in the future just create a new file when we decide to write test cases for `forecastCompiler` class

Comment (peterdudfield) [reply tests/test_forecast_compiler.py:7]: that would be great

Comment (zaryab-ali) [reply tests/test_forecast_compiler.py:7]: just made a commit with this update, please have a look

**Review by peterdudfield — APPROVED** [review]

**Comment by peterdudfield** [comment]
Thanks so much for this, happy for me to merge?

**Comment by zaryab-ali** [comment]
Yes please, go ahead
